apps/node/src/app/main/events/__init__.py

```diff
@@ -66,12 +66,18 @@ def route_requests(message, socket):
     if isinstance(message, bytearray):
         return forward_binary_message(message)
 
+    request_id = None
     try:
         message = json.loads(message)
+        request_id = message.get(MSG_FIELD.REQUEST_ID)
```

Comment:
maybe having a `message.get(request_id,None)` and later check if the request actually supplied it? So we don't want to throw error if the user didn't supply the request id

Reply:
Yes, it's optional. `.get()` returns None by default not throwing error when field is missing from dict.
Then there's a check if request_id was supplied:
https://github.com/OpenMined/PyGrid/pull/730/files#diff-a7f092ad3c6a3a9977cbc9d3a7c960eaR77-R78